tests/fixtures/tutorial-real-input/README.md

```
@@ -330,3 +330,35 @@ $ node scripts/run-real-input-preview.mjs --fixture bad-slug --out /tmp/x
 | `tests/helpers/realInputFixtureRegistry.cjs` | Registry loading, fixture lookup, path resolution, file validation |
 | `scripts/validate-real-input-preview-artifact.cjs` | Contract validation |
 | `tests/helpers/realInputSmokeCoverageReport.cjs` | Coverage report generation |
+
+## CI CLI Smoke
+
+In addition to the E2E smoke test (which runs both fixtures via Jest), CI also
+executes the operator-facing CLI directly for one fixture to prove end-to-end
+orchestration works outside the test harness:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)